README.md

```diff
@@ -15,7 +15,7 @@ With immudb you can track changes in sensitive data in your transactional databa
 tamperproof immudb database. This allows you to keep an indelible history of sensitive data, for example debit/credit card transactions.
 <img align="right" src="img/immudb-mascot-small.png" width="256px"/>
 
-Traditional transaction logs are hard to scale and are mutable. So there is no way to know for sure if your data has been compromised.
+Traditional DB transactions and logs are hard to scale and are mutable. So there is no way to know for sure if your data has been compromised.
```

Comment:
The original wording seems ok. It refers transaction log file on an ACID dbms.

Reply:
@Gjergj This change was specifically requested as such.